rosplan_planning_system/launch/includes/chimp_planner_interface.launch

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
There is no PANDA planner launch - does it use the same as CHIMP?

Reply:
The integration with PANDA and in general with HDDL planners is still work under progress. At least Lilotane and PANDA are planned. The KB already accepts HDDL formalism which is a step in the right direction.